tests/publiccloud/ipa.pm

```diff
@@ -16,6 +16,22 @@ use strict;
 use warnings;
 use testapi;
 
+sub extract_startup_timings {
+    my $string = shift;
+    my $res    = {};
+    $string =~ s/Startup finished in\s*//;
+    $string =~ s/=(.+)$/+$1 (overall)/;
+    for my $time (split(/\s*\+\s*/, $string)) {
+        if ($time =~ /((\d{1,2})min\s*)?(\d{1,2}\.\d{1,3})s\s*\((\w+)\)/) {
```

Comment:
I think we miss here the `\s*`
```suggestion
        if ($time =~ /((\d{1,2})\s*min\s*)?(\d{1,2}\.\d{1,3})s\s*\((\w+)\)/) {
```
Maybe it's not worth to do a run with the latest code.

Reply:
why ? systemd-analyze returning timings always like this `1min`

Reply:
if the format is `Xmin` without space, let's leave it as it is. If we see problems later on, we can always fix it. Merging it now.